### DISCUSSION TITLE
[question] docker update question

### DISCUSSION THREAD

**Discussion by Rouzax** [zwave-js/zwave-js-ui#2329] [General]
I know this has nothing to do with ZwaveJS2MQTT 😄, but I'm trying to figure out what the best way is to update the Docker image I have running of Zwavejs2MQTT with Docker Compose. I'll admit this is the first service I have running in Docker so kind of a nitwit here.
Have looked through the https://zwave-js.github.io/zwavejs2mqtt/#/getting-started/docker but wasn't able to find it.

**Comment by kpine** [comment]
Use a version tag, like `zwavejs/zwavejs2mqtt:6.5.2`, and docker-compose will automatically pull the image when you change the tag and bring up the app (`docker-compose up -d`). If you use `latest`, then you need to manually issue a [pull](https://docs.docker.com/compose/reference/pull/), e.g. `docker-compose pull && docker-compose up -d`.

FYI, there's a [Discussions page](https://github.com/zwave-js/zwavejs2mqtt/discussions) for questions.
